app/views/content/train-to-be-a-teacher/promos/_events-near-you.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      caption: "24 May 2022",
+      heading: "Virtual Train to Teach event",
```

Comment:
@EllieNodder Is it too long a title to have 'National Virtual Train to Teach event'?

Trying to emphasise it is for everyone.....

Reply:
@gemmadallmandfe felt like national virtual was a bit of a mouthful, and the fact that it's virtual should already emphasise it is for everyone (but equally happy to add it in)

Reply:
@EllieNodder Fair shout - let's leave as is